tests/test_source_sampling.py

```diff
@@ -23,6 +23,38 @@
 def try_rm_file(filename):
     return lambda: os.remove(filename) if os.path.exists(filename) else None
 
+@with_setup(None, try_rm_file('tet.h5m'))
+def test_single_tet_tag_names_map():
+    """This test tests uniform sampling within a single tetrahedron. This is
+    done by dividing the tetrahedron in 4 smaller tetrahedrons and ensuring
+    that each sub-tet is sampled equally.
+    """
+    seed(1953)
+    m = Mesh(structured=True,
+             structured_coords=[[0, 3, 3.5], [0, 1], [0, 1]],
+             mats = None)
+    m.src = IMeshTag(2, float)
+    m.src[:] = [[2.0, 1.0], [9.0, 3.0]]
+    e_bounds = np.array([0, 0.5, 1.0])
+    m.bias = IMeshTag(2, float)
+    m.bias[:] = [[1.0, 2.0], [3.0, 3.0]]
+    m.mesh.save("sampling_mesh.h5m")
+    filename = "sampling_mesh.h5m"
+
+    # right condition
+    tag_names = {"src_tag_name": "src",
```

Comment:
I don't think this line is needed anymore - the next line replaces it.

Reply:
This line has been removed.